tests/vm_test.go

```diff
@@ -1101,7 +1101,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					Expect(err).ToNot(HaveOccurred())
 					Expect(newVM.Spec.RunStrategy).ToNot(BeNil())
 					Expect(*newVM.Spec.RunStrategy).To(Equal(v1.RunStrategyHalted))
-					Expect(len(newVM.Status.StateChangeRequests)).To(Equal(0))
+					Expect(newVM.Status.StateChangeRequests).To(HaveLen(0))
```

Comment:
```suggestion
					Expect(newVM.Status.StateChangeRequests).To(BeEmpty())
```
is more idiomatic

Reply:
I prefer to handle it in a different PR, this one focuses on one thing and it is much easier for one to review it.

And BTW, I am not really sure it is valid to do such a replacement with `BeEmpty` blindly, as I believe it should only be done for cases where the type is a slice or map, but not for other things (e.g. a string).

Reply:
Ok. I don't mind waiting with `BeEmpty`

Reply:
Feel free to vote :)